README.md

```diff
@@ -195,6 +198,19 @@ async fn handle_event(
 }
 ```
 
+and in your `Cargo.toml`, include the following dependencies:
+
+```toml
+...
```

Comment:
This seems to render as a syntax error to the user since it's not valid TOML, just leave it out

Reply:
:+1: